New version: py-requests 2.26.0

iarspider opened the pull request.

iarspider committed:
New version: py-requests 2.26.0

iarspider committed:
New version: py-requests 2.26.0

wdconinc committed:
[geant4] new version geant4@10.7.3 (#27568)

* [geant4] new version geant4@10.7.3

Release notes: https://geant4-data.web.cern.ch/ReleaseNotes/Patch4.10.7-3.txt, no build system changes.

* [geant4] depends_on geant4-data@10.7.3

* [geant4-data] new version @10.7.3

alalazo committed:
Make CUDA and ROCm architecture conditional  (#27185)

* Make CUDA and ROCm architecture conditional

fixes #14337

The variant to specify which architecture to use
for CUDA and ROCm are now conditional on +cuda and
+rocm respectively.

* cp2k: make all CUDA related variants conditional on +cuda

adamjstewart commented on var/spack/repos/builtin/packages/py-requests/package.py:
```suggestion
    depends_on('py-charset-normalizer@2.0.0:2.0', type=('build', 'run'), when='@2.26.0: ^python@3:')
```

adamjstewart commented on var/spack/repos/builtin/packages/py-requests/package.py:
This is now only required for Python 2

adamjstewart commented on var/spack/repos/builtin/packages/py-requests/package.py:
```suggestion
    depends_on('py-idna@2.5:2', type=('build', 'run'), when='@2.26.0: ^python@:2')
```

adamjstewart commented on var/spack/repos/builtin/packages/py-requests/package.py:
```suggestion
    depends_on('py-idna@2.5:2', type=('build', 'run'), when='@2.23:2.25')
```
Otherwise this constraint will be applied to all versions

iarspider committed:
Apply suggestions from code review

Co-authored-by: Adam J. Stewart <ajstewart426@gmail.com>

iarspider committed:
Apply suggestions from code review

Co-authored-by: Adam J. Stewart <ajstewart426@gmail.com>

iarspider committed:
Update package.py

iarspider committed:
Update package.py

alalazo committed:
raja: fix recipe for old versions (#27591)

Raja didn't depend on camp prior to v0.12.0

haampie committed:
suite-sparse package: add support for blas symbol suffix (#27510)

luszczek committed:
Intel packages: add support for LLVM OpenMP (#26517)

rblake-llnl committed:
ncurses: fix external package detection on OS X (#27490)

iarspider committed:
New version: py-singledispatch 3.7.0 (#27584)

* New version: py-singledispatch 3.7.0

* Apply suggestions from code review

Co-authored-by: Adam J. Stewart <ajstewart426@gmail.com>

* Update package.py

Co-authored-by: Adam J. Stewart <ajstewart426@gmail.com>

manuelakuhn committed:
py-zope-event: add 4.5.0 (#27595)

iarspider committed:
New version: py-stevedore 3.5.0 (#27586)

nhanford committed:
bugfix: Allow legacy tests to be read after hash break (#26078)

* added a test case

Co-authored-by: Nathan Hanford <hanford1@llnl.gov>

lrknox committed:
Add hdf5-vol-external-passthrough package. (#27289)

* Add hdf5-vol-external-passthrough package.

* Add version v1.0 for vol-external-passthrough release.

* Remove incorrect comment.

* Add source url and checksum.  Update HDF5 minimum version.

lrknox committed:
Update to latest commit hash for vol-log-based. (#27612)

Update minimum hdf5 version to 1.13.0.

manuelakuhn committed:
py-cython: add 3.0.0a9 (#27609)

* py-cython: add 3.0.0a9

* Update var/spack/repos/builtin/packages/py-cython/package.py

Co-authored-by: Adam J. Stewart <ajstewart426@gmail.com>

Co-authored-by: Adam J. Stewart <ajstewart426@gmail.com>

alalazo committed:
Remove support for Python 2.6 (#27256)

Modifications:
- [x] Removed `centos:6` unit test, adjusted vermin checks
- [x] Removed backport of `collections.OrderedDict`
- [x] Removed backport of `functools.total_ordering`
- [x] Removed Python 2.6 specific skip markers in unit tests
- [x] Fixed a few minor Python 2.6 related TODOs in code

Updating the vendored dependencies will be done in separate PRs

vsoch committed:
adding new package go cosign (#27606)

Signed-off-by: vsoch <vsoch@users.noreply.github.com>

Co-authored-by: vsoch <vsoch@users.noreply.github.com>

manuelakuhn committed:
py-flask: add 2.0.2 (#27615)

haampie committed:
Fix leaky tests (#27616)

* fix: cc.py should use a function not session scope
* fix: don't let build env vars leak to other tests
* fix: don't leak build env in dev_build test

wohlbier committed:
Add encryption library packages: helib, palisade-development, and seal (#27495)

Add packages for three common homomorphic encryption libraries.
HElib, Palisade, and Seal. Also add package for number theoretic library NTL.

haampie committed:
llvm: use ninja by default (#27521)

* llvm: use ninja by default

* Use ninja for omp when it's not a runtime

elfprince13 committed:
Racket package: disable parallel build; add variants (#27506)

- Prevent `-j` flags to `make`, which has been known to cause problems
  with Racket builds.
- Add variants for various common build flags, including support
  for both versions of the Racket VM environment.

In addition:

- Prefer the minimal release to improve install times. Bells and
  whistles carry their own runtime dependencies and should be
  installed via `raco`. An enterprising user may even create a
  `RacoPackage` class to make spack aware of `raco` installed packages.
- Match the official version numbering scheme.

srekolam committed:
rocm-tensile: Add new GPUs to tensile-architecture list (#26913)

vsoch committed:
Adding oras go package (#27605)

Signed-off-by: vsoch <vsoch@users.noreply.github.com>

Co-authored-by: vsoch <vsoch@users.noreply.github.com>

haampie committed:
libblastrampoline: use tarballs and add maintainer (#27619)

* Use tarballs for libblastrampoline

* Add myself as maintainer

alalazo committed:
Update six to v1.16.0 (#27265)

alalazo committed:
Update Jinja2 to v2.11.3 and MarkupSafe to v1.1.1 (#27264)

albestro committed:
umpire: fix gcc@10.3.0 conflict (#27620)

Make sure that `gcc@10.3.0-[number]` also matches the conflict.

iarspider commented:
@adamjstewart can you merge this PR manually? The pipeline fails for an unrelated reason...

iarspider committed:
Merge branch 'update-py-requests' of github.com:iarspider/spack into update-py-requests
